editor/src/core/workers/parser-printer/parser-printer-arbitrary-elements.spec.tsx

```diff
@@ -87,19 +89,22 @@ export var ${BakedInStoryboardVariableName} = (props) => {
                 firstChild.elementsWithin[Object.keys(firstChild.elementsWithin)[0]]
               expect(getJSXAttribute(elementWithin.props, 'data-uid')).not.toBeNull()
             } else {
-              fail('First child is not an arbitrary block of code.')
+              throw new Error('First child is not an arbitrary block of code.')
             }
           } else {
-            fail('Root element not a JSX element.')
+            throw new Error('Root element not a JSX element.')
           }
         } else {
-          fail('Not a component at the root.')
+          throw new Error('Not a component at the root.')
         }
       },
-      (unparsed) => fail(unparsed),
+      (unparsed) => {
+        throw new Error(JSON.stringify(unparsed))
+      },
       parseResult,
     )
   })
+  // eslint-disable-next-line jest/expect-expect
```

Comment:
what does this do? why is it disabled?

Reply:
it expects an expect, but there's no expect here because this test calls a helper function which in turn calls expect :)

Reply:
Well that's not at all what I expected